flink-ai-flow/ai_flow/client/ai_flow_client.py

```diff
@@ -96,7 +96,7 @@ def __init__(self,
         ModelCenterClient.__init__(self, server_uri)
         MetricClient.__init__(self, server_uri)
         SchedulerClient.__init__(self, server_uri)
-        self.enable_ha = False
+        self.aiflow_ha_enabled = False
```

Comment:
Don't modify the variable name of the `enable_ha`.

Reply:
It should be modified because the parent class `NotificationClient` has a same variable `enable_ha`. The `enable_ha` in child class would override parent's. It's unexpected.